### PR TITLE
fix(objectviewer): adjust with guidelines

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The objectviewer styles were not specified

**What is the new behavior?**

it's has been specified and here is the changes done (spec is under tfd)

**Does this PR introduce a breaking change?**

- [x] No

screen shot 2017-03-06 at 14 46 23

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by frassinier** [comment]
How did you extract these values since we don't have any PSD file?

File: packages/components/src/ObjectViewer/Table/Table.scss [hunk 1]
```diff
@@ -1,11 +1,23 @@
 .table {
+	td {
+		line-height: 25px;
+	}
+
 	&:global(.table-hover) >tbody > tr:hover {
 		font-weight: initial;
 	}
 
 	&:global(.table) > thead > tr {
```
Comment (jsomsanith) [inline packages/components/src/ObjectViewer/Table/Table.scss]: Missing size. In the storybook it says 14px, in guideline 12px

File: packages/components/src/ObjectViewer/Table/Table.scss [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
-		background-color: $brand-primary;
+		background-color: $dove-gray;
 		color: $white;
-		font-weight: bold;
+		font-family: 'Source Sans Pro';
+		text-transform: uppercase;
+		font-weight: 900;
```
Comment (jsomsanith) [inline packages/components/src/ObjectViewer/Table/Table.scss]: Missing value in guideline, but seems bolder than in the guideline

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

File: packages/components/src/ObjectViewer/Table/Table.component.js [hunk 3]
```diff
@@ -19,11 +19,26 @@ function getAbsolutePath(index, key, flat) {
 	return `$[${index}]['${key}']`;
 }
 
+function getHeaders(keys, isFlat) {
```
Comment (acateland) [inline packages/components/src/ObjectViewer/Table/Table.component.js]: unit test would be great

File: packages/components/src/ObjectViewer/Table/Table.scss (hunk 2 above)
Comment (acateland) [inline packages/components/src/ObjectViewer/Table/Table.scss]: beware could be anti aliasing

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://227.talend.surge.sh)

**Pull request merged**